### DISCUSSION TITLE
/v2/zh/guide/get-started/intro.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#206] [Announcements]
# /v2/zh/guide/get-started/intro.html

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/guide/get-started/intro.html

**Comment by yufire** [comment]
6666

**Reply by yufire** [reply]
6666

**Reply by yufire** [reply]
# 6666

**Comment by chenwenjingchaojidashuaige** [comment]
作者你好，这个主题在移动端（手机浏览器中）有个Bug，点击右上角打开导航栏，无法滚动。
